pkg/api/validation/validation.go

```diff
@@ -1338,6 +1339,23 @@ func ValidateReplicationControllerUpdate(oldController, controller *api.Replicat
 	allErrs := errs.ValidationErrorList{}
 	allErrs = append(allErrs, ValidateObjectMetaUpdate(&controller.ObjectMeta, &oldController.ObjectMeta).Prefix("metadata")...)
 	allErrs = append(allErrs, ValidateReplicationControllerSpec(&controller.Spec).Prefix("spec")...)
+	allErrs = append(allErrs, ValidateReplicationControllerStatus(&controller.Status).Prefix("status")...)
```

Comment:
This is an internal helper function that is used by both 

```
ValidateReplicationController(...)
ValidateReplicationControllerStatusUpdate(...)
```

to verify just the rc.status section of the resource.

the ```ValidateReplicationControllerStatusUpdate``` function (which I added in this PR) is what you expect.